### ISSUE TITLE
[Term]: ods:roleOrder

### ISSUE THREAD

**Issue by ben-norton** [DiSSCo/openDS#185]
### Term Name

ods:roleOrder

### Digital Object Name

Agent

### Feedback

The purpose, criteria, and scope are unclear. Also, how should two agents with equal but different roles be ordered?

**Comment by DavidFichtmueller** [comment]
The way I would have interpreted it, is that this property indicates the order among agents with the same role, e.g. when there are multiple collectors and one being the primary one. So that for they are displayed in the correct order, when shown to a user in any kind of UI. Agents with a different role, would be grouped by that role in their respective order. But you are right, this is more of an assumption and not directly stated in the definition, so the definition should be extended here. 

See also  #153 .

**Comment by samleeflang** [comment]
Yes, David's comment is correct. We will extend the definition. If there is a clear primary and secondary collector, identifier, etc... this property can be used to indicate who should be shown first. It might be a bit of a stretch at the moment, but it won't hurt to have the possibility in there.
